stable/consul-esm/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+appVersion: "0.4.0"
+description: Consul ESM for external service health checks in Consul
+name: consul-esm
+version: 0.3.1
+home: https://github.com/hashicorp/consul-esm
```

Comment:
any suggestion on `appVersion` and `version` ? `appVersion` we use the consul-esm docker tag ? and for version separate sever ?

Reply:
Yes